### PR TITLE
Fix issues for section contents

### PR DESCRIPTION
**Pull request by ShobhitAd**
Fixes issues found after validation for section contents(rpc/enum/struct parameter tables, json examples, sequence diagrams, etc.)

### CODE REVIEW

File: docs/SDL/GetListOfPermissions/index.md [hunk 1]
```diff
@@ -66,8 +66,10 @@ GetListOfPermissions provide  External UCS Status to HMI
 ![GetListOfPermissions](./assets/GetListOfPermissionsInformingHMI.png)
 |||
 
+|||
```
Comment (iCollin) [inline docs/SDL/GetListOfPermissions/index.md]: you sure this should have format changed? check https://smartdevicelink.com/en/guides/hmi/sdl/getlistofpermissions/

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (iCollin) [inline .vscode/settings.json]: should be removed

File: docs/SDL/GetListOfPermissions/index.md (hunk 1 above)
Comment (ShobhitAd) [reply docs/SDL/GetListOfPermissions/index.md]: Resolved in https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/183/commits/f878b97b51a2def86fb730215446dac8ecba24c4

File: .vscode/settings.json (hunk 2 above)
Comment (ShobhitAd) [reply .vscode/settings.json]: Nice catch. Removed in https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/183/commits/88c9319ad5cb3b91bab20e3891cba1233e78b246

**Pull request merged**